using-mobile-extensions/adobe-target/README.md

```diff
@@ -115,30 +115,34 @@ Prefetched offer content does not persist across launches. The prefetch content
 {% tab title="Android" %}
 #### Java
 
-#### Using the `TargetPrefetch` Builder
 
-The `TargetPrefetch` builder helps create a `TargetPrefetch` instance with the specified data. The returned instance can be used with `prefetchContent`, which accepts a `TargetPrefetch` object list to prefetch offers for the specified mbox locations.
+#### Using the `TargetPrefetch` Constructor
+
+Using `TargetPrefetch` Constructor, you can create a `TargetPrefetch` instance with the specified data. It currently accepts location name and an optional `TargetParameters` object. The returned instance can be used with `prefetchContent`, which accepts a `TargetPrefetch` object list to prefetch offers for the specified mbox locations. 
```

Comment:
- Does "Constructor" have to be capitalized?
- Please reword the 2nd sentence so that it read as follows: "It currently accepts the location name and an optional `TargetParameters` object."

Reply:
Changed "Constructor" to "constructor"